README_ZHT.md

```diff
@@ -0,0 +1,399 @@
+                     airplane
+
+    00:36 < tester> man sites take so long to load on i2p
+    00:36 < tester> i value speed over anonymity any day
+
+    <DuoNoxSol> it's notably more reliable than the normal internet
+
+    09:46 < Kubuxu> I so love cjdns code base
+
+    <whyrusleeping> my internet is way better now.
+    <whyrusleeping> thanks
+    <whyrusleeping> i'm really upset and sad that its better
+    <whyrusleeping> but also quite happy
+
+    21:01 <@grewalsat> this is amazing. with my workpalce speedtest.net results I get around 6+mb speed, and with my cjdns-gate as vpn network I'm getting like 11-15mb download speed in speedtest.net
+    21:01 <@grewalsat> :P
+    21:01 <@grewalsat> plus, access anything! :D
+
+    <davidar> Yeah, I have to admit I sort of avoided hypeirc because of stuff like that
+
+## 社群
+
+* [irc://irc.efnet.org/#cjdns][IRC Web]
+* [Hyperboria][] 目前最大的 cjdns 網路，至2015年10月止有2100個節點。
+* [Project Meshnet][]
+* [/r/darknetplan][]
+* [#cjdns on Twitter][]
+
+
+## 相關文件
+
+* [Project Goals](doc/projectGoals.md)
+* [Cjdns Whitepaper](doc/Whitepaper.md)
+* [Cjdns on Wikipedia][]
+
+進階配置（英文）:
+
+* [Setup a cjdns NAT gateway for your LAN](doc/nat-gateway.md)
+* [Install cjdns on OpenIndiana](doc/open-indiana.md)
+
+感謝您所貢獻的時間與精力
+
+cjdns 開發人員 敬上
+
+--------------------------------------------------------------------------------
+
+## 如何安裝 cjdns
+
+這些步驟以 Linux Debian 系統與 OS X 系統為主，
+相關概念在其他系統上應該足以通用（不過請不要期待可以逐句貼上）。
+
+### 0. 安裝相關程式
+
+在任意系統安裝 [Node.js](http://nodejs.org/)，非必要但強烈建議。
+如無 Node.js 或版本錯誤，安裝程式會自動下載安裝至 Source Tree 中。
+
+#### Debian 系統:
+
+    sudo apt-get install nodejs git build-essential python2.7
+
+#### Fedora 22+ 系統:
+
+    sudo dnf install install nodejs git
+    sudo dnf install @development-tools
+
+#### RHEL 系統 (會新增 EPEL 的 Repository):
+
+    sudo yum localinstall https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+    sudo yum install install nodejs git
+    sudo yum install @development-tools
+
+#### OS X 系統:
+
+利用 homebrew 安裝:
+
+    brew install cjdns
+
+#### OpenBSD 系統:
+
+非常遺憾目前 OpenBSD 還在實驗階段。
+
+    pkg_add git node gcc gmake bash
+
+選擇 gcc-4.8.1p2 之後版本。
+
+#### FreeBSD 系統:
+
+所有所需相關程式已預先在 FreeBSD' ports 編譯完成。
+
+    pkg install gmake node
+
+#### Arch 系統:
+
+你可以用以下指令安裝 cjdns
+
+    pacman -S cjdns
+
+如果你需要從原始碼重新編譯，所有需要的程式可以用以下方式安裝
+
+    pacman -S nodejs git base-devel
+    
+另外，你可能會想利用 AUR 安裝包裝 `cjdns-git`。
+安裝完成後，配置檔在 `/etc/cjdroute.conf`。
+用以下指令啟動 `cjdns.service`：
+
+        systemctl start cjdns
+
+停止指令：
+
+       systemctl stop cjdns
+
+### 1. 從 GitHub 取得 cjdns 
+
+從 Github 複製 Repository 並切換至下載檔案資料夾：
+
+    git clone https://github.com/cjdelisle/cjdns.git cjdns
+    cd cjdns
+
+### 2. 初始化
+
+    ./do
+
+當你看見 `Build completed successfully, type ./cjdroute to begin setup.`，
+再繼續以下步驟:
+
+--------------------------------------------------------------------------------
+
+## 安裝
+
+執行 cjdroute 並不要顯示 HELP 選項：
```

Comment:
The English is a little ambiguous here and the incorrect interpretation was translated here.

>Run cjdroute without options for HELP:

Should really read "In order to display the HELP menu, run cjdroute without options". So the Chinese should be something like:

>顯示 HELP 選項, please 執行 cjdroute without specifying anything